scico/examples.py

```diff
@@ -242,18 +242,20 @@ def tile_volume_slices(x: JaxArray, sep_width: int = 10) -> JaxArray:
     return out
 
 
-def create_cone(img_shape: Shape, center: Optional[list] = None):
+def create_cone(img_shape: Shape, center: Optional[list[float]] = None) -> Array:
     """Compute a 2D map of the distance from a center pixel.
 
     Args:
-        img_shape : Shape of the image for which the distance map is being computed.
-        center : Tuple of center pixel ids. If None, this is set to the center of the image
+        img_shape: Shape of the image for which the distance map is being
+            computed.
+        center: Tuple of center pixel ids. If ``None``, this is set to
+            the center of the image.
 
     Returns:
-        An image containing a 2D map of the distances
+        An image containing a 2D map of the distances.
     """
 
-    if center == None:
+    if center is None:
         center = [img_dim // 2 for img_dim in img_shape]
```

Comment:
Is the intent here really to only support integer center values? If not, this should be `... img_dim / 2 ...`.

Reply:
Yes, I originally thought of only integer values. But making it floats (/ instead of //) would be more general and should work.